web/satellite/src/components/team/AddUserPopup.vue

```diff
@@ -294,6 +302,12 @@ export default class AddUserPopup extends Vue {}
         font-family: 'montserrat_regular' !important;
         font-size: 16px;
         line-height: 25px;
+
+        a {
+            color: #2683FF;
+            text-decoration: none;
```

Comment:
Do not remove underline decoration from links, unless it's an actual button. It's an important signifier for color-blind people.